server/queue/queue-api/src/test/java/org/apache/james/queue/api/ManageableMailQueueContract.java

```diff
@@ -90,7 +93,9 @@ default void dequeueShouldDecreaseQueueSize() throws Exception {
     default void noAckShouldNotDecreaseSize() throws Exception {
         enQueue(defaultMail().name("name").build());
 
-        Flux.from(getManageableMailQueue().deQueue()).blockFirst().done(false);
+        Flux.from(getManageableMailQueue().deQueue())
+            .doOnNext(Throwing.consumer(item -> item.done(false)))
```

Comment:
`sneakyThrow`?

Reply:
For what? It is a test, wrapping don't add much and if I can avoid the sneakythrow dark magic cheating then I prefer it...